recipes/bonjour/system/conanfile.py

```diff
@@ -0,0 +1,26 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class BonjourSystemConan(ConanFile):
+    name = "bonjour"
+    version = "system"
+    description = "Conan package for Apple's Bonjour"
+    topics = ("Bonjour", "DNS-SD", "mDNS")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://developer.apple.com/bonjour/"
+    license = "Apache-2.0", "BSD-3-Clause"
+    settings = "os", "arch", "compiler", "build_type"
+
+    def validate(self):
+        if self.settings.os != "Windows" or tools.cross_building(self):
```

Comment:
Typo?

```suggestion
        if self.settings.os == "Windows" or tools.cross_building(self):
```

Reply:
I think maybe it's just that `choco` isn't installed? The condition is right.